utils/global-state-update-gen/src/generic/config.rs

```diff
@@ -31,16 +31,18 @@ pub struct AccountConfig {
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ValidatorConfig {
     pub bonded_amount: U512,
-    pub delegation_rate: u8,
-    pub delegators: Vec<DelegatorConfig>,
+    pub delegation_rate: Option<u8>,
```

Comment:
Shouldn't this be a required field? I don't think you can create a validator on MainNet without this.

Reply:
I see you resolved it, but just for the sake of anyone stumbling upon this - `None` will just mean "leave this unchanged". (Or, if there was no previous value, it would default to 0.)